src/txn/txn.c

```diff
@@ -1215,7 +1221,7 @@ __wt_txn_rollback(WT_SESSION_IMPL *session, const char *cfg[])
 			 */
 			if (F_ISSET(txn, WT_TXN_PREPARE))
 				WT_RET(__wt_txn_resolve_prepared_op(
-				    session, op, false));
+				    session, op, false, NULL));
```

Comment:
Any reason why we disable this logic for rollback. We have this diagnostic code to ensure that we don't miss any prepared update in our search, which can happen for rollback as well.